include/industry_standard/spdm.h

```diff
@@ -478,6 +478,76 @@ typedef struct {
 #define SPDM_REGISTRY_ID_IANA_CBOR 0xa
 #define SPDM_REGISTRY_ID_MAX  0xa
 
+typedef struct {
+    uint8_t id;
+    uint8_t vendor_id_len;
+    /* uint8_t vendor_id[vendor_id_len]; */
+} spdm_svh_header_t;
+
+typedef struct {
+    uint8_t id; /* SPDM_REGISTRY_ID_DMTF */
```

Comment:
Can use `spdm_svh_header_t` since that's already defined.

Reply:
yes.